libraries/AP_HAL/RCOutput.cpp

```diff
@@ -0,0 +1,46 @@
+#include "AP_HAL.h"
+
+extern const AP_HAL::HAL &hal;
+
+// helper function for implementation of get_output_mode_banner
+const char* AP_HAL::RCOutput::get_output_mode_string(enum output_mode out_mode) const
+{
+    // convert mode to string
+    switch (out_mode) {
+    case MODE_PWM_NONE:
+    default:
```

Comment:
Any reason to have `default:`?

If someone adds an output mode it would be nice for the compiler to tell them they had to add stuff to this function!

Reply:
No good reason in this case so removed.  I sometimes add it because I've found that if the value somehow ends up outside the enum range (which is possible with parameters cast to an enum) the default is the way to catch it.  Still, this doesn't apply in this case so I've removed it in a follow up commit. txs.